src/page/Home.tsx

```diff
@@ -52,6 +59,168 @@ const EXAMPLE_DATA_GRCh38 = [
+                    <td>HGVSc</td>
+                    <td>
+                        <Link
+                            to={`/variant/7:g.140453136A>T?transcriptId=ENST00000288602`}
+                        >
+                            ENST00000288602.6:c.1799T{'>'}A
+                        </Link>
+                    </td>
+                </tr>
+                <tr>
+                    <td>rs id</td>
+                    <td>
+                        <Link to={`/variant/7:g.140453136A>T`}>
+                            rs113488022
+                        </Link>
+                    </td>
+                </tr>
+            </tbody>
+        </Table>
+        <strong>More examples:</strong>
+        <div>
+            <Link to={`/variant/5:g.1295228G>A`}>5:g.1295228G{'>'}A</Link>
+        </div>
+        <div>
+            <Link to={`/variant/7:g.55249071C>T`}>EGFR p.T790M</Link>
+        </div>
+        <div>
+            <Link to={`/variant/17:g.7577121G>A`}>TP53 R273C</Link>
+        </div>
+        <div>
+            <Link
+                to={`/variant/17:g.41223015_41223016del?transcriptId=ENST00000478531`}
+            >
+                BRCA1:c.68_69del
+            </Link>
+        </div>
+        <div>
+            <Link to={`/variant/17:g.41276045_41276046del`}>rs397509106</Link>
+        </div>
+        <div style={{ marginTop: 10 }}>
+            For more information, see documentation{' '}
+            <a href="https://docs.genomenexus.org" target="_top">
+                here
+            </a>
+            .
+        </div>
+    </>
+);
+
+const searchExample = (
+    <>
+        <strong>Examples</strong>:{' '}
+        <Link to={`/variant/7:g.55249071C>T`}>EGFR:p.T790M</Link>,{' '}
+        <Link to={`/variant/7:g.140453136A>T`}>7:g.140453136A{'>'}T</Link>,{' '}
+        <Link to={`/variant/17:g.41276045_41276046del`}>rs397509106</Link>,{' '}
+        <Link to={`/variant/17:g.7577121G>A`}>TP53 R273C</Link>
+    </>
+);
+
+const searchInstructionPopover = (hideOverlay: any) => (
+    <Popover
+        id="popover-basic"
+        placement="right"
+        title={
+            <>
+                <span>How to search on Genome Nexus</span>
+                <i
+                    className="fas fa-times-circle"
+                    style={{
+                        color: 'black',
+                        cursor: 'pointer',
+                        float: 'right',
+                    }}
+                    onClick={() => {
+                        hideOverlay();
+                    }}
+                ></i>
+            </>
+        }
+        style={{ maxWidth: 'none' }}
+    >
+        {searchInstructionContent}
+    </Popover>
+);
+
+const SearchInstruction = () => {
+    let overlay: any;
+    return (
+        <OverlayTrigger
+            ref={(ref: any) => (overlay = ref)}
+            trigger="click"
+            placement="right"
+            overlay={searchInstructionPopover(() => overlay.hide())}
```

Comment:
Something like this would be better:

```suggestion
            overlay={<SearchInstructionPopover onClose={() => overlay.hide()} />}
```

Reply:
@onursumer For this one if I do component syntax, `searchInstructionPopover` will not be able to locate the info button and appears on top left instead
![image](https://user-images.githubusercontent.com/16869603/129593828-3ef5e82b-a83a-487b-ad5b-2b6a1077e528.png)

Reply:
Hmm, how about like this? Still same error?
```suggestion
            overlay={() => <SearchInstructionPopover onClose={() => overlay.hide()} />}
```

Reply:
@onursumer yes unfortunately. It becomes an individual component above everything